README.md

```diff
@@ -139,6 +139,48 @@ Query test can be done like this:
 
 For more detail take a look at [wiki file](https://github.com/HTTP-APIs/http-apis.github.io/blob/master/hydra-agent-redis-graph.md)
 
+#### Querying Redis
```

Comment:
I started this session with some important information, mainly the naming of labels and alias. I.e: If you want to find Drones by their label you have to:

`GRAPH.QUERY apidoc "MATCH (p:objectsDrone) RETURN p"` **p:Drone wouldn't work.**

I'll adjust, complement and write it better while I progress. For now, it's mainly for the sake of documenting as I go and I think it shouldn't even be on the main readme.md, perhaps a readme.md inside the Redis folder because it's mainly for our development use.

Reply:
we store the ApiDoc in the graph?

Reply:
No, apidoc is the name for the Graph that Sandeep used in his code, basically the Redis key that has "graphdata" as its type. I guess it's because he started implementing having the apidoc in mind and later they changed... I guess this should also be changed to avoid confusion?

Reply:
I suggest to change the name to `apigraph` or we can get confused